core/spec/models/spree/reimbursement_spec.rb

```diff
@@ -19,6 +19,7 @@
         let(:reimbursement) { Spree::Reimbursement.new(number: nil) }
 
         before do
+          allow(reimbursement).to receive(:recalculate_order)
```

Comment:
This method was stubbed because the reimbursement in this spec does not receive the order, so the reimbursement creation failed.
Now I'm going to check that order is provided before run recalculate on it.